content/docs/workflow-examples/escalation/_index.md

```diff
@@ -1,4 +1,4 @@
 ---
-title: "Escalation Workflow Examples"
+title: "Escalation"
```

Comment:
Do you think we should end each workflow title in this section with " Workflow" suffix? 
e.g. `Escalation Workflow` (not sure if this is redundant or more clear).